depthai_examples/ros1_src/yolov4_spatial_publisher.cpp

```diff
@@ -51,9 +56,17 @@ dai::Pipeline createPipeline(bool syncNN, bool subpixel, std::string nnPath, int
     colorCam->setInterleaved(false);
     colorCam->setColorOrder(dai::ColorCameraProperties::ColorOrder::BGR);
 
-    monoLeft->setResolution(dai::MonoCameraProperties::SensorResolution::THE_400_P);
+    if(resolution == "720p"){
+        monoResolution = dai::MonoCameraProperties::SensorResolution::THE_720_P; 
+    }else if(resolution == "400p" ){
+        monoResolution = dai::MonoCameraProperties::SensorResolution::THE_400_P; 
+    }else{
+        monoResolution = dai::MonoCameraProperties::SensorResolution::THE_800_P; 
+    }
```

Comment:
can you validate if `800p` is passed on the string too ? and throw an error on else if it has an invalid string. 
And there is 480p res too (Which is for OAK-D-LITE) we can add that too here.

Reply:
Fixed in b9b6b81.